README.md

```diff
@@ -1,3 +1,7 @@
+[![codecov](https://codecov.io/gh/nt4f04uNd/sweyer/branch/master/graph/badge.svg)](https://codecov.io/gh/nt4f04uNd/sweyer)
+
+# Sweyer
```

Comment:
I don't like how it looks - the current README is perfect to me, so let's just not add this badge anywhere

Reply:
Hm, I would like to have a link to the coverage statistics, and I think it would also be helpful to someone who wants to participate. How about we move the badge into the `CONTRIBUTING.md` and I add a short paragraph about coverage?

Reply:
Yes, `CONTRIBUTING.md` would be a good place for it